salt/_renderers/metalk8s_kubernetes.py

```diff
@@ -38,7 +38,7 @@ def _step_name(manifest, absent=False):
     try:
         name = manifest['metadata']['name']
     except KeyError:
-        raise SaltRenderError('Object `metadata.name` must be set.')
+        raise SaltRenderError('Object `metadata.name` must be set.')  # pylint: disable=raise-missing-from
```

Comment:
This one could have been kept (just to know whether the missing key is `metadata` or `name`)

Reply:
Right, make sense